tinydb/__init__.py

```diff
@@ -8,8 +8,8 @@
 
 Usage example:
 
->>> from tinydb import TinyDB, where
->>> from tinydb.storages import MemoryStorage
+>>> from . import TinyDB, where
+>>> from .storages import MemoryStorage
```

Comment:
I think we should keep the absolute import here as that's what users will usually continue to use. Or am I mistaken?